src/styleguide/routes/forms.vue

```diff
@@ -109,7 +109,10 @@
 
         <e-fieldset legend="Date Pickers">
           <eDate v-model="form.date" name="date" />
-          <cDatePicker :start.sync="form.date" name="date-picker" @change="onDatePickerChange" />
+          <cDatePicker :start.sync="form.startDate"
+                       :end.sync="form.endDate"
+                       range
```

Comment:
todo: please ALWAYS define attributes without a value AFTER dynamic and static attributes but before event handlers. Maybe the value was removed by ESLint if you use `:range="true"` because it is the same as the attribute name itself. But be aware of edge cases when defining props with "multi-type-support". See https://eslint.vuejs.org/rules/prefer-true-attribute-shorthand.html